lib/utils/utils.js

```diff
@@ -12,6 +12,21 @@ module.exports = {
     return !methods || methods.length === 0;
   },
 
+  getAssignmentsToThis: function(node) {
+    return _.filter(node.value.body.body, (item) => {
```

Comment:
Ah, so you mean just do
`return item.type === 'ExpressionStatement' && item.expression.type === 'AssignmentExpression' && ...`

Yep a little more concise, thanks!